<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogRedaction</class>
 <widget class="QDialog" name="dialogRedaction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>281</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelID">
         <property name="text">
          <string>Enter ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Enter the name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelProducer">
         <property name="text">
          <string>Enter the producer:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelArticle">
         <property name="text">
          <string>Enter the article:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSum">
         <property name="text">
          <string>Enter the sum of products:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelWeight">
         <property name="text">
          <string>Enter the weight of 1 product:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPrice">
         <property name="text">
          <string>Enter the price of 1 product:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditID"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditProducer"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditArticle"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditSum"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditWeight"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPrice"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonOk">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
